### PR TITLE
Feat: Add LDIO File out module

### PR DESCRIPTION
**Pull request by Tomvbe**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -17,6 +17,7 @@
         <module>ldi-core</module>
         <module>ldi-orchestrator</module>
         <module>ldi-nifi</module>
+        <module>ldi-orchestrator/ldio-connectors/ldio-file-out</module>
```
Comment (Yalz) [inline pom.xml:20]: Wrong place to add the submodule 😉

**Review by Yalz — COMMENTED** [review]
So this component only work if a timestampPath is present?

**Comment by Tomvbe** [comment]
> So this component only work if a timestampPath is present?

Yes, we need a timestamp to determine the order of the members in order to reconstruct the LDES in the correct order when reading members from the archive (next story).

**Comment by Yalz** [comment]
> > So this component only work if a timestampPath is present?
> 
> Yes, we need a timestamp to determine the order of the members in order to reconstruct the LDES in the correct order when reading members from the archive (next story).

Can we include the option to default to the current timestamp? So that this works for basic Linked Data without a timestamp?

**Review by Yalz — APPROVED** [review]
Missing docs though ;)

**Pull request merged**